src/test/kotlin/org/rucca/cheese/client/UserClient.kt

```
@@ -0,0 +1,113 @@
+        val request =
+            """
+            {
+                "username": "$username",
+                "password": "$password"
+            }
+        """
+        val response = target.request().post(Entity.entity(request, MediaType.APPLICATION_JSON))
+        val result = JSONObject(response.readEntity(String::class.java))
+        try {
+            return result.getJSONObject("data").getString("accessToken")
+        } catch (e: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)